### ISSUE TITLE
The tr and tr tags created by jqgrid have roles and these tags should not have roles

### ISSUE THREAD

**Issue by julie4754** [tonytomov/jqGrid#990]
The tr and tr elements created by jqgrid have roles and these elements should not have roles according to WCAG as these already have clear purposes. This may be a hangover from when jqgrids were created on div elements rather than table elements .

**Comment by julie4754** [comment]
Rule: Table structure elements cannot specify an explicit 'role' within table containers
